src/journal-remote/meson.build

```diff
@@ -1,12 +1,14 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
 systemd_journal_upload_sources = files(
+        'journal-remote-util.c',
```

Comment:
Hm, that's my fault, but using `journal-remote-util.c` in journal-upload sounds slightly spurious. Maybe, better to rename `journal-compression-util.c` or so.